### PR TITLE
My cloud haiku

### PR DESCRIPTION
**Pull request by hajko**

### CODE REVIEW

File: _haikus/SasSy.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: SaaSy
+author: Julian
+---
+
+"No software" the goal
```
Comment (ltagliaferri) [inline _haikus/SasSy.md]: Make sure you have two spaces at the end of each line of the haiku, or else use the `<br>` HTML tag to break the line.

File: _haikus/SasSy.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: SaaSy
+author: Julian
+---
+
+"No software" the goal  
+Salesforce evangelized
```
Comment (ltagliaferri) [inline _haikus/SasSy.md]: This line needs one more syllable.

File: _haikus/SasSy.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: SaaSy
+author: Julian
+---
+
+"No software" the goal  
+Salesforce evangelized 
+Now everywhere
```
Comment (ltagliaferri) [inline _haikus/SasSy.md]: This line also needs one more syllable.

**Comment by hajko** [comment]
I initially tried
 +"No software" the goal  
+Salesforce evangelized it
+Now it's everywhere  But the automated testing said I had one too many syllables in the second and third lines so I removed syllables.
I don't think it will let me submit a pull request if it doesn't pass the automated testing, right?
Julian
   On Saturday, October 27, 2018, 3:33:43 PM PDT, ltagliaferri <notifications@github.com> wrote: 
 
 
@ltagliaferri requested changes on this pull request.

Thanks for submitting this haiku! We still need some more syllables in the haiku for it to follow the format. Please update this and we'll merge in :)

In _haikus/SasSy.md:
> @@ -0,0 +1,9 @@
+---
+layout: haiku
+title: SaaSy
+author: Julian
+---
+
+"No software" the goal  

Make sure you have two spaces at the end of each line of the haiku, or else use the <br> HTML tag to break the line.

In _haikus/SasSy.md:
> @@ -0,0 +1,9 @@
+---
+layout: haiku
+title: SaaSy
+author: Julian
+---
+
+"No software" the goal  
+Salesforce evangelized 

This line needs one more syllable.

In _haikus/SasSy.md:
> @@ -0,0 +1,9 @@
+---
+layout: haiku
+title: SaaSy
+author: Julian
+---
+
+"No software" the goal  
+Salesforce evangelized 
+Now everywhere  

This line also needs one more syllable.

—
You are receiving this because you authored the thread.

Reply to this email directly, view it on GitHub, or mute the thread.

| 
| 
| 
|  |  |

 |

 |
| 
|  | 
My cloud haiku by hajko · Pull Request #53 · do-community/cloud_haiku


 |

 |

 |





| 
| 
| 
|  |  |

 |

 |
| 
|  | 
Build software better, together

GitHub is where people build software. More than 28 million people use GitHub to discover, fork, and contribute ...
 |

 |

 |

**Comment by ltagliaferri** [comment]
We can override it if it doesn't pass the checks but it's a correct haiku. The software is not perfect, but our team is contributing to that to try to improve it. But yes, your original haiku is 💯 if you can revert back to that, we'll override and merge. Thank you :)

**Comment by hajko** [comment]
Ok, I just edited and resubmitted. Thanks.
   On Saturday, October 27, 2018, 4:43:46 PM PDT, ltagliaferri <notifications@github.com> wrote: 
 
 
We can override it if it doesn't pass the checks but it's a correct haiku. The software is not perfect, but our team is contributing to that to try to improve it. But yes, your original haiku is 💯 if you can revert back to that, we'll override and merge. Thank you :)

**Comment by beardicus** [comment]
Thanks for sticking with it! Sorry our automated testing complicated things unnecessarily. We're working on it!

**Pull request merged**